subsys/bluetooth/mesh/brg_cfg_srv.c

```diff
@@ -111,12 +111,12 @@ static int bridging_table_remove(const struct bt_mesh_model *model, struct bt_me
 	struct bt_mesh_bridging_table_entry entry;
 	uint8_t status = STATUS_SUCCESS;
 
+	entry.directions = 0;
```

Comment:
The PTS failed because of the netkey_check was true so passing the if statement. The remove was then done. At that moment the directions is a random number so the test failed when receiving directions with the value 0x04. When it is removed the direction should be 0.

Reply:
I could have solved this by doing "struct bt_mesh_bridging_table_entry entry = { 0 };"